rfcs/text/0000-application-monitoring-for-ga.md

```diff
@@ -0,0 +1,125 @@
+### DCP PR:
+
+# System monitoring for General Availability
+
+## Summary
+
+This RFC describes the first standard revision for monitoring the distributed systems of the Human Cell Atlas (HCA) Data Coordination Platform (DCP). The standard is scoped to the minimum viable set of monitoring infrastructure the DCP will need to reach General Availability (GA). We anticipate that future revisions will be needed to meet our Service Level Objectives (SLOs) as DCP matures.
+
+The term monitoring is used here as defined in the [Google Site Reliability Engineering (SRE)](https://landing.google.com/sre/sre-book/chapters/monitoring-distributed-systems/) program. This document borrows heavily from SRE.
+
+## Author(s)
+
+* [Matthew Weiden](mailto:mweiden@chanzuckerberg.com)
+
+## Shepherd
+
+
+## Motivation
+
+Monitoring the distributed systems [enables operators](https://landing.google.com/sre/sre-book/chapters/monitoring-distributed-systems/#why-monitor-pWsBTZ) to analyze long-term trends in system performance, compare experiments against a baseline, alert on failures, and conduct retrospective analysis on systems (debugging). These abilities are critical to keeping software correct and available.
+
+### User Stories
+
+* As a DCP operator, I want to compare experiments against a baseline, so that I can compare design alternatives.
+* As a DCP operator, I want to conduct retropsective analysis on systems, so that I can debug systems.
+* As a DCP operator, I want to analyze long-term trends in system performance, so that I can estimate changes in cost and performance.
+* As a DCP operator, I want to be alerted of system failure, so that I know when something goes wrong and can keep the system available for our users.
```

Comment:
Are we leaving the definition of system failure to each component? As it stands the health checks don't really check for 'system failure' or 'available for our users'. Is this a suggestion to change the nature/triggers of alerts by GA?